### PR TITLE
Merge #7192: Unify product name to as few places as possible

### PR DESCRIPTION
**Pull request by kyuupichan**
027fdb8 When/if the copyright line does not mention Bitcoin Core developers, add a second line to copyrights in -version, About dialog, and splash screen (Luke Dashjr)
cc2095e Rewrite FormatParagraph to handle newlines within input strings correctly (Luke Dashjr)
cddffaf Bugfix: Include COPYRIGHT_HOLDERS_SUBSTITUTION in Makefile substitutions so it gets passed to extract-strings correctly (Luke Dashjr)
29598e4 Move PACKAGE_URL to configure.ac (Luke Dashjr)
78ec83d splashscreen: Resize text to fit exactly (Luke Dashjr)
3cae140 Bugfix: Actually use _COPYRIGHT_HOLDERS_SUBSTITUTION everywhere (Luke Dashjr)
4d5a3df Bugfix: gitian-descriptors: Add missing python-setuptools requirement for OS X (biplist module) (Luke Dashjr)
e4ab5e5 Bugfix: Correct copyright year in Mac DMG background image (Luke Dashjr)
917b1d0 Set copyright holders displayed in notices separately from the package name (Luke Dashjr)
c39a6ff Travis & gitian-osx: Use depends for ds_store and mac_alias modules (Luke Dashjr)
902ccde depends: Add mac_alias to depends (Luke Dashjr)
82a2d98 depends: Add ds_store to depends (Cory Fields)
de619a3 depends: Pass PYTHONPATH along to configure (Cory Fields)
e611b6e macdeploy: Use rsvg-convert rather than cairosvg (Luke Dashjr)
63bcdc5 More complicated package name substitution for Mac deployment (Luke Dashjr)
1a6c67c Parameterise 2009 in translatable copyright strings (Luke Dashjr)
d5f4683 Unify package name to as few places as possible without major changes (Luke Dashjr)

### CODE REVIEW

**Comment by kyuupichan** [comment]
I'd llike to merge this as I keep running into conflicts caused by it in other situations.  Arguably it's an improvement too.  Getting this done was quite painful....

To check I've done a make deploy and installed the binary on a Mac.  The binary is working, the "About Bitcoin Unlimited" dialog looks good, as does the RPC console and bitcoin-cli help.  I've not checked anything else.

There might be objections around the attempt(s) I've made with the copyright attribution in order to keep the patch close to the original.  I'm happy to do followups if necessary to address those.

**Comment by kyuupichan** [comment]
Oh, and make check was fine of course.

File: Makefile.am [hunk 1]
```diff
@@ -34,9 +41,9 @@ WINDOWS_PACKAGING = $(top_srcdir)/share/pixmaps/bitcoin.ico \
   $(top_srcdir)/share/pixmaps/nsis-wizard.bmp \
   $(top_srcdir)/doc/README_windows.txt
 
-OSX_PACKAGING = $(OSX_DEPLOY_SCRIPT) $(OSX_FANCY_PLIST) $(OSX_INSTALLER_ICONS) $(OSX_BASE_LPROJ_DIR) \
-  $(top_srcdir)/contrib/macdeploy/$(OSX_BACKGROUND_IMAGE) \
-  $(top_srcdir)/contrib/macdeploy/DS_Store \
+OSX_PACKAGING = $(OSX_DEPLOY_SCRIPT) $(OSX_FANCY_PLIST) $(OSX_INSTALLER_ICONS) \
+  $(top_srcdir)/contrib/macdeploy/$(OSX_BACKGROUND_SVG) \
```
Comment (gandrewstone) [inline Makefile.am:45]: looks like this is going to change the background image file.  Have you verified that it show a BU image?

Comment (kyuupichan) [reply Makefile.am:45]: The startup splashscreen still shows the infinity and the words bitcoin unlimited if that's what you're referring to.

Comment (kyuupichan) [reply Makefile.am:45]: Hold on, was running a different branch

Comment (kyuupichan) [reply Makefile.am:45]: No, just rebuilt, splashscreen is good - looks same to me with infinity and BU wording

**Comment by gandrewstone** [comment]
note rerun autogen.sh when you pull this PR

**Pull request merged**